.travis.yml

```diff
@@ -40,5 +40,6 @@ matrix:
     - env: TOXENV=py27-flake8
     - env: TOXENV=py34-flake8
     - env: TOXENV=docstrings
+    - env: TOXENV=readme
```

Comment:
You could have kept `-env: TOXENV=docs` too. Or you can send a new PR with that :-D